definitions/private-law/json/CaseEventToFields/ManageDocuments/CaseEventToFields.json

```diff
@@ -125,7 +125,8 @@
     "PageID": 1,
     "PageDisplayOrder": 1,
     "PageFieldDisplayOrder": 1,
-    "ShowSummaryChangeOption": "Y"
```

Comment:
Please move this to first field in the page

Reply:
@shashisk97  as discussed there are old manage document fields which are present on this page which I believe need to be cleaned up, will check with @yogendraU  if we can remove those as part of this.

Reply:
Hi @shashisk97 , unfortunately, we cant remove older ManageDocuments event since we have data against those events in production. Thats we have given this old event authorisation as "R" in AuthorisationCaseEvent.json